server/preprocessing/other-scripts/test/base-snapshot.R

```diff
@@ -0,0 +1,40 @@
+rm(list = ls())
+
+library(rstudioapi)
+library(arsenal)
+
+options(warn=1)
+
+wd <- dirname(rstudioapi::getActiveDocumentContext()$path)
+
+setwd(wd) #Don't forget to set your working directory
+
+query <- "digital education" #args[2]
+service <- "base"
+params <- NULL
+params_file <- "snapshot_params_base.json"
```

Comment:
I don't think the above paramters are really needed. They are usually passed to get_papers.

Reply:
Yes, I intentionally left them in to have a reference on what queries the snapshots were created - but we can document that in a different way and remove the parameters.